src/index.html

```diff
@@ -16,17 +16,34 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="page">
+    <form
+      class="search-bar search-bar-big"
+      data-qa="big"
+    >
+      <label class="search-bar__label search-bar__lable-big">
+        <input
+          data-qa="keypress"
+          class="search-bar__input search-bar__input-big"
+          type="text"
+          placeholder="Try “Los Angeles“"
+        />
+      </label>
+    </form>
+
+    <form
+      class="search-bar search-bar-small"
+      data-qa="small"
+    >
+      <label class="search-bar__label search-bar__lable-small">
```

Comment:
There is a typo in the class name 'search-bar__lable-small'. It should be 'search-bar__label-small'.